app/src/main/java/at/irfc/app/ui/map/MapScreen.kt

```diff
@@ -0,0 +1,30 @@
+package at.irfc.app.ui.map
+
+import androidx.compose.foundation.background
+import androidx.compose.foundation.layout.*
+import androidx.compose.material3.MaterialTheme
+import androidx.compose.material3.Text
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.unit.dp
+import at.irfc.app.presentation.map.MapViewModel
+import com.ramcosta.composedestinations.annotation.Destination
+import org.koin.androidx.compose.getViewModel
+
+@Composable
+@Destination
+fun MapScreen(vm: MapViewModel = getViewModel()) {
```

Comment:
```suggestion
fun MapScreen() {
```

Probably no viewModel needed for this screen.